terraform/charm/large_deployment/variables.tf

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+variable "cluster_name" {
+  type        = string
+  description = "The cluster name of the fleet."
+  default     = "opensearch"
+}
+
+variable "main" {
```

Comment:
nit: this variable would be more clear as "main-ochestrator"

Reply:
Fair, @phvalguima any idea for a name here?

Reply:
I could be wrong but couldn't we call it `main cluster manager`?

Reply:
`cluster_manager` has a meaning in opensearch, it is a role, but it commonly refers to the "elected primary"

Reply:
@MiaAltieri don't look at me, I am terrible choosing names, lol.

Maybe calling it "main_orchestrator", which is a good ref. to our own docs.